opentelemetry_collector/receiver/metricgenerator/metric_generator.go

```diff
@@ -103,9 +124,49 @@ func MakeSingleValueDistributionTimeSeries(
 		Buckets:               MakeBuckets([]float64{val}, bounds),
 	}
 
+	return makeTimeseriesFromDistribution(&distribution, startTime, currentTime, labels)
+}
+
+func GetSumOfSquaredDeviationFromIntDist(
+	sum, sumSquares, count int64) float64 {
```

Comment:
nit: keep function signature in a single line, here and everywhere else.

Reply:
I've moved this one to one line, but for some of the other functions that would result in an excessively long line.